scripts/release/params.py

```diff
@@ -0,0 +1,94 @@
+import sys
+
+import requests
+
+QUAY_REPOS = {
+    "IMAGES_APISERVER":  "ds-pipelines-api-server",
+    "IMAGES_ARTIFACT":  "ds-pipelines-artifact-manager",
+    "IMAGES_PERSISTENTAGENT":  "ds-pipelines-persistenceagent",
+    "IMAGES_SCHEDULEDWORKFLOW":  "ds-pipelines-scheduledworkflow",
+    "IMAGES_MLMDENVOY":  "ds-pipelines-metadata-envoy",
+    "IMAGES_MLMDGRPC":  "ds-pipelines-metadata-grpc",
+    "IMAGES_MLMDWRITER":  "ds-pipelines-metadata-writer",
+    "IMAGES_DSPO":  "data-science-pipelines-operator",
+}
+
+ARCH = "amd64"
+
+
+def fetch_quay_repo_tag_digest(quay_repo, quay_org, tag):
+    api_url = "https://quay.io/api/v1/repository/{0}/{1}/tag/?specificTag={2}".format(
```

Comment:
f-strings plz

Reply:
^above applies to most string construction throughout these scripts, as IMO its much cleaner and maintainable than `.format`